doc/build-windows.md

```diff
@@ -33,10 +33,10 @@ Windows](https://msdn.microsoft.com/en-us/commandline/wsl/install_guide).
 Full instructions to install WSL are available on the above link.
 To install WSL on Windows 10 with Fall Creators Update installed (version >= 16215.0) do the following:
 
-1. Enable the Windows Subsystem for Linux feature (Run OptionalFeatures.exe)
-  * Select Windows Subsystem for Linux
-  * Click OK
-  * Restart if necessary
+1. Enable the Windows Subsystem for Linux feature
+  * Open the Windows Features dialog (`OptionalFeatures.exe`)
+  * Enable 'Windows Susbsystem for Linux (Beta)'
```

Comment:
Just FYI,  it's out of beta now :)

![image](https://user-images.githubusercontent.com/6394033/35187819-220b2c2e-fe22-11e7-8a2c-775859a83094.png)

Reply:
Ah yes that's true! Did that yesterday 😄